main.tf

```diff
@@ -8,15 +8,16 @@
  * While waiting for updates on that repo you will need to use a special fork and branch:
  *
  * ```sh
- * git clone git@github.com:chrisgilmerproj/bucket-antivirus-function.git
- * git checkout cg_all_prs
+ * git clone git@github.com:upsidetravel/bucket-antivirus-function.git
+ * cd bucket-antivirus-function
+ * git checkout master
```

Comment:
This whole block do we even need these instructions since we aren't using a special fork anymore?

Reply:
yeah, you always have to build. Just wanted to make it clear how to get the repo:)